tests/examples/requirements-minimal/output/expected/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
```

Comment:
Maybe split this comment into two lines if possible? I would also suggest to add a comment at the very top of the script.

Reply:
I wanted to do the same, unfortunately I don't know how to do this with tomlkit. There is only one comment associated with the line. If I add a manual newline, when reading it back tomlkit no longer associates the second line with the same comment, so each run of dfg just keeps adding another copy of the second line. If you're more familiar with the package and know how this might be accomplished any help would be appreciated!

Re: top of file, I didn't want to do that because there are many parts of this file that are dependencies (and are therefore not auto-generated). I did not want to confuse developers editing those components.

Reply:
Sorry, I'm not familiar enough with tomlkit to be able to know that from the top of my head. As a guess, you probably to add a separate `Comment` node instead of adding the comment to the existing node.

I was thinking more of a disclaimer at the top, like `; Parts of this file were auto-generated, please see the inline notes ...`.

Reply:
Unfortunately I didn't find a good way to address this request yet. I'll play with it later if I can, but for the moment I think it's worth moving forward with the existing changeset and we can add that later if we find a way to do so.

Reply:
That's totally fine.